README.md

```diff
@@ -181,50 +239,54 @@ vault.fetch(Cat.class)
 ```
 
-The above example creates an `Observable` that subscribes and observes on the same thread initiating the query, so make sure to change that according to your requirements. A typical use-case would be:
+The above example creates an `Observable` that subscribes and observes on the same thread initiating the query. That changes if this typical use-case is used:
+
 ```java
 vault.observe(Cat.class)
     .all()
     .subscribeOn(Schedulers.io())
     .observeOn(AndroidSchedulers.mainThread())
 ```
 
-### Migrations
+Migrations
+----------
 
-Whenever changes are introduced to any of the previously used models, a migration has to be applied. Simply increment the version number for your space to trigger a migration:
+Whenever changes are introduced to any of the previously used Models, a migration has to be applied. Increment the version number to trigger a migration:
 
 ```java
 @Space(value = "cfexampleapi", models = { Cat.class }, dbVersion = 2)
 public class DemoSpace { }
 ```
 
-Note: this will delete any previously persisted data.
+> Note: this will delete any previously persisted data and reaquires them.
 
-### Database pre-seeding
+Preseeding
+----------
 
-Depending on the amount of content in a given space, initial synchronization might take some time. For that we've added support to pre-seed the database with static content. 
+Depending on the amount of content in a given space, initial synchronization might take some time. For that support to pre-seed the database with static content got added. 
 
-If you want to create an initial database file, please use the `VaultDatabaseExporter`. This class takes an Android Context and a Vault Space like you would provide Vault with. If you call the exporters `.export(..)` method, it will create a sqlite database in `src/main/assets/initial_seed.db`. Feel free to rename this file to your needs. If you want to tell Vault to use this database, please create a Vault space class like this:
+For creating an initial database file, use the `VaultDatabaseExporter`. This class takes an Android Context and a Vault Space. Calling the `.export(..)` method, it  creates a sqlite database in `src/main/assets/initial_seed.db`. Instrucing Vault to use this is done like so:
 
 ```java
 @Space(
-    value = "foo", // space id of the space to use
-    models = { Bar.class },  // model classes to be used
+    value = "{spaceid}", // space id of the space to use
+    models = { Cat.class },  // model classes to be used
     copyPath = "initial_seed.db" // name of the just created database file.
 )
 public class VaultSpace { }
 ```
 
-If you want to update this database file, you could think about running a [robolectric](robolectric.org) test before releasing. This test would sync Contentful data to the existing database.
+Udating this database file is done by leveraging a [robolectric](robolectric.org) test before releasing. This test would sync Contentful data to the existing database.
```

Comment:
The database file is updated by leveraging a [robolectric](robolectric.org) test before releasing. This test syncs Contentful data to the existing database.